packages/synapse-react-client/src/components/FeaturedDataTabs/QueryPerFacetPlotsCard.tsx

```diff
@@ -29,9 +33,7 @@ export function getQueryRequest(
     concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
     partMask:
       SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
-      SynapseConstants.BUNDLE_MASK_QUERY_FACETS |
-      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
-      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
```

Comment:
We didn't need this much data to render the plots (and I was able to figure this out with type-safety with the new generic capabilities added to `QueryContext`)